client/asset/eth/eth.go

```diff
@@ -260,6 +325,10 @@ type baseWallet struct {
 	walletsMtx sync.RWMutex
 	wallets    map[uint32]*assetWallet
 
+	monitoredTxsMtx sync.RWMutex //nolint:structcheck
```

Comment:
What's the `nolint` for?  I'm not seeing anything from the `unused` linter, although `structcheck` is disabled with Go 1.19 so I can't check that readily.

Reply:
`structcheck` was saying that this field is unused, but it's not true. Maybe because it's not used in any baseWallet methods.